terraform/environments/core-logging/route53_resolver_logs.tf

```diff
@@ -15,7 +15,7 @@ data "aws_iam_policy_document" "route53-query-logging-policy" {
       "logs:PutLogEvents",
     ]
 
-    resources = ["arn:aws:logs:*:*:log-group:/aws/route53/*"]
+    resources = ["arn:aws:logs:*:*:log-group:route53-query-logging-policy"]
```

Comment:
Looking at the available log groups in the core-logging account, I think you wanted to use: `modernisation-platform-r53-resolver-logs` log group.

so it would be something like this
`resources = ["arn:aws:logs:*:*:log-group:modernisation-platform-r53-resolver-logs:*"]`

However, I think it would be better if you narrow it down to that account as well, so this is not a complete answer, but a pointer

Reply:
@ewastempel - I think a condition that includes only accounts in our OU might be the best way to do this.